docs/features/multiple-networks/multi-homing.md

```diff
@@ -1,33 +1,99 @@
 
-## Configuring secondary networks
 To allow pods to have multiple network interfaces, the user must provide the
 configurations specifying how to connect to these networks; these
-configurations are defined in a CRD named `NetworkAttachmentDefinition`.
+configurations are set in a CRD named `NetworkAttachmentDefinition`, defined by
+the [Kubernetes Network Custom Resource Definition De-facto Standard](https://github.com/k8snetworkplumbingwg/multi-net-spec).
+
+Notes:
+
+* layer 2 and layer 3 topologies are overlays - thus, they do not need **any** previous physical network configuration.
+
+## Prerequisites
+
+- [multus-cni](https://github.com/k8snetworkplumbingwg/multus-cni)
+
+## Motivation
+
+Multihoming is essential when you need more than one network interface on your pods. This can be useful for various
+use cases, such as virtual network functions (VNFs), firewalls, or virtualization (virt) where the default
+cluster network might not be suitable. For example, multihoming allows for live migration of pods without changing
+their IP addresses, which is often not possible with the primary network.
+
+In OVN-K, multihoming supports several virt-specific features. These
+include [persistent IP addresses for virtualization](#persistent-ip-addresses-for-virtualization-workloads) workloads,
+ensuring that VMs retain their IP addresses even when they move across nodes. This enhances workload mobility and
+minimizes disruptions.
+
+Multihoming is also compatible with the multi-network policy API, which can provide further security rules on the
+traffic.
 
-Below you will find example attachment configurations for each of the current
-topologies OVN-K allows for secondary networks.
+## User-Stories
+
+- As a Cluster-Admin, I want to configure secondary networks for specific pods so that I can enable
+  specialized/sensitive workloads with distinct network requirements.
+- As a Cluster-Admin, I want to facilitate seamless live migration of VMs within so that I can maintain established TCP
+  connections and preserve VM IP configurations during migration.
+
+## User-Cases
+
+- cluster-wide overlay network on layer 2:
+  In this case example, two VMs from different namespaces - VMA and VMC - are connected over a secondary-network.
+  VMB is not exposed to this traffic.
+  ![multi-homing-use-case-layer2](multi-homing-use-case-layer2.png)
+- cluster-wide overlay network on localnet:
+  In this case example, Pod and VM workloads accessing a relational DB reachable via the physical network (i.e. deployed
+  outside Kubernetes).
+  ![multi-homing-use-case-localnet](multi-homing-use-case-localnet.png)
+
+## How to enable this feature on an OVN-Kubernetes cluster?
+
+This feature is enabled by default on all OVN-Kubernetes clusters. You don't need to do anything extra to start using
+this feature. There is a Feature Config option `enable-multi-network` under `OVNKubernetesFeatureConfig` config that can
+be used to disable this feature.
```

Comment:
Hm, I think this is wrong ....

I mean, it is true for OpenShift, but not upstream.

You need to start the kind cluster w/ the proper flag - `--multi-network-enable` . 

Right ?

Reply:
Does `How to enable this feature on an OVN-Kubernetes cluster?` == how do I enabled it using the kind.sh script?

Reply:
This is wrong. You need to say something like
```suggestion
The `multi-network` feature must be enabled in the OVN-Kubernetes configuration. Please use the `Feature Config` option `enable-multi-network` under `OVNKubernetesFeatureConfig` config to enable it.
```

Reply:
DONE